ui/apps/platform/src/Components/ContainerVolumesInfo.tsx

```diff
@@ -40,7 +40,7 @@ function ContainerVolumeInfo({ volumes }: ContainerVolumeInfoProps) {
                             <StackItem key={volume.name}>
                                 <ExpandableSection
                                     toggleText={volume.name}
-                                    onToggle={(_e, v) => setToggleAtIndex(v)}
+                                    onToggle={() => setToggleAtIndex(index)}
```

Comment:
This sets the toggle state in an **array** of items, so we need the index of the item in the array instead of the boolean value from the toggle event of the expandable section.